kolibri/core/assets/src/views/auth-message.vue

```diff
@@ -36,19 +36,16 @@
     },
     computed: {
       defaultDetails() {
-        return this.$tr('mustBeSignedInAsRole', {
-          role: this.$tr(this.authorizedRole),
-        });
+        return this.$tr(this.authorizedRole);
       },
     },
     $trs: {
-      admin: 'an Admin',
-      adminOrCoach: 'an Admin or Coach',
+      admin: 'You must be signed in as an Admin to view this page',
+      adminOrCoach: 'You must be signed in as an Admin or Coach to view this page',
+      learner: 'You must be signed in as a Learner to view this page',
+      registeredUser: 'You must be signed in to view this page',
+      superuser: 'You must be a Superuser to view this page',
```

Comment:
What about 'You must have Superuser permissions to view this page'?

Reply:
I second the motion!